src/components/navigation-top/navigation-top.jsx

```diff
@@ -19,11 +21,11 @@ export function NavigationTop(){
                     </div>
                     <Link to="/wishlist" className="heart-icon">
                         <i className="fa-solid fa-heart"></i>
-                        <span className="heart-badge"></span>
+                        <span className="heart-badge">{wishlistState.wishlist.length >0 ? wishlistState.wishlist.length : ''}</span>
```

Comment:
Could you try to have a badge where there is a round background and a number in between ? 
would be much easier to look at , currently the colors merge together creating a less than perfect UI experience.